### DISCUSSION TITLE
I am getting this error again

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1950] [Q&A]
![image](https://github.com/user-attachments/assets/a33829ab-b7e4-4463-a33f-7cb02cd4220c)

**Comment by chaithanya1812** [comment]
### Hi @Nazrina412 , In you code you are simply using {{ token }}--> wrong

![image](https://github.com/user-attachments/assets/d0060d55-f733-44ce-a24e-586f982b6cf1)
